server/service/osquery_utils/queries.go

```diff
@@ -129,7 +130,17 @@ var hostDetailQueries = map[string]DetailQuery{
 				return nil
 			}
 
-			if rows[0]["major"] != "0" || rows[0]["minor"] != "0" || rows[0]["patch"] != "0" {
+			if rows[0]["name"] == "Ubuntu" {
+				// Ubuntu takes a different approach to updating patch IDs so we instead use
+				// the version string provided after removing the code name
+				regx := regexp.MustCompile(`\(.*\)`)
+				vers := regx.ReplaceAllString(rows[0]["version"], "")
+				host.OSVersion = fmt.Sprintf(
+					"%s %s",
+					rows[0]["name"],
+					strings.TrimSpace(vers),
+				)
```

Comment:
@gillespi314 generally this LGTM, but one question before moving forward: from the linked issue seems like it would be acceptable to use the full version string like so:

```go
fmt.Sprintf("%s %s", rows[0]["name"], rows[0]["version"])
```

which seems to make things easier for us long term, wdyt?

Reply:
@roperzh, I initially implemented it that way and then noticed that when it is rendered in the UI the Ubuntu versions will most often get truncated if the release code name is included. I also noted that we seem to not include release code names for other platforms.